src/EvoSC.Common/Database/Models/Player/DbPlayerSettings.cs

```diff
@@ -7,10 +8,28 @@ namespace EvoSC.Common.Database.Models.Player;
 public class DbPlayerSettings : IPlayerSettings
 {
     public const string TableName = "PlayerSettings";
-    
-    [Column]
-    public long PlayerId { get; set; }
-    
-    [Column]
-    public string DisplayLanguage { get; set; }
+
+    [Column] public long PlayerId { get; set; }
+
+    [Column] public string DisplayLanguage { get; set; }
+
+    [Column] public string? HiddenManialinks { get; set; }
+
+    public List<string> GetHiddenManialinks()
+    {
+        return HiddenManialinks != null
+            ? HiddenManialinks.Split(",").ToList()
+            : [];
+    }
+
+    public void SetHiddenManialinks(List<string> hiddenManialinks)
+    {
+        if (hiddenManialinks.IsNullOrEmpty())
+        {
+            HiddenManialinks = null;
+            return;
+        }
+
+        HiddenManialinks = string.Join(",", hiddenManialinks);
+    }
```

Comment:
The way I've done this before is using property setters and getters.

```suggestion
    [Column(nameof(HiddenManialinks))]
    public string? DbHiddenManialinks { get; set; }
    
    public string? HiddenManialinks  {
         get => DbHiddenManialinks != null ? DbHiddenManialinks.Split(",").ToList()  : [];
         set {
               if (hiddenManialinks.IsNullOrEmpty())
              {
                  DbHiddenManialinks = null;
                  return;
              }

              DbHiddenManialinks = string.Join(",", value);
         }
    }
```

It then follow the same pattern as everything else; set a property, save to db.